packages/rlc-common/test/helpers/nameUtils.spec.ts

```diff
@@ -103,6 +103,17 @@ describe("#normalizeName", () => {
     });
   });
 
+  describe("for method", () => {
+    it("should return the name with prefix $ for the method name is a reserved name", () => {
+      expect(normalizeName("continue", NameType.Method, true)).to.equal(
+        "$continue"
+      );
+      expect(normalizeName("break", NameType.Method, true)).to.equal(
+        "$break"
+      );
```

Comment:
@v-jiaodi pls help add more UTs.

Reply:
add missing cases in spector : https://github.com/microsoft/typespec/pull/7413

Reply:
could @v-jiaodi try to enableOperationGroup as false to see if it works for special words case?

Reply:
@qiaozha we would have generation issue then and create an issue to track this https://github.com/Azure/autorest.typescript/issues/3206!